extensions/agroal/runtime-dev/src/main/java/io/quarkus/agroal/runtime/dev/ui/DatabaseInspector.java

```diff
@@ -5,20 +5,9 @@
 import java.io.UncheckedIOException;
 import java.net.URI;
 import java.net.URISyntaxException;
-import java.sql.Connection;
-import java.sql.DatabaseMetaData;
+import java.sql.*;
```

Comment:
I think wildcard imports are forbidden. See https://github.com/quarkusio/quarkus/blob/main/CONTRIBUTING.md#ide-config-and-code-style : 

> Next, disable wildcard imports: navigate to Editor -> Code Style -> Java -> Imports and set Class count to use import with '*' to 999. Do the same with Names count to use static import with '*'.

You can try `./mvnw process-sources` to run automatic formatting, but I'm not sure that will solve the wildcard import issue; you might need your IDE to solve that.

Reply:
Yeah, it feels like I already did that multiple times but maybe IJ loses the config 🤔 `process-sources` does not fix that indeed, should be fixed now.